boards/nucleo_l412rb_p.json

```diff
@@ -0,0 +1,46 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32L412xx",
+    "f_cpu": "80000000L",
+    "mcu": "stm32l412rb",
+    "product_line": "STM32L412xx",
+    "variant": "NUCLEO_L412RB_P"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32L412RB",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_board": "st_nucleo_l4",
+    "openocd_target": "stm32l4x",
+    "svd_path": "STM32L4x2.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "cmsis",
+    "mbed",
+    "stm32cube",
+    "libopencm3",
+    "zephyr"
```

Comment:
It seems that this board can be used only with low-level frameworks `cmsis` and `stm32cube`.

Reply:
Zephyr should follow soon: https://github.com/zephyrproject-rtos/zephyr/pull/33976 :) Not sure how easy it is to add for the other platforms